pages/1.13/security/oss/authentication/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: layout.pug
+navigationTitle:  Authentication
+title: Authentication
+excerpt: Authenticating users against DC/OS
+menuWeight: 30
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+# Authentication in DC/OS
+
+In DC/OS user authentication is enabled by default.
```

Comment:
Should we say "required by default"? I think this might be the actual message we want to emit here.